exercises/hello-world/hello_world.go

```diff
@@ -1,6 +1,7 @@
 // This is a "stub" file.  It's a little start on your solution.
 // It's not a complete solution though; you have to write some code.
 
+// Package hello says hello to the world.
```

Comment:
How about:

> Package hello should have a package comment that summarizes what it's about.

That would both pass linting and suggest that people should write their own comment.